.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+	"semi": true,
+	"singleQuote": true,
+	"bracketSpacing": true,
+	"arrowParens": "always",
+	"trailingComma": "none",
+	"useTabs": true
```

Comment:
nickpick: I fancy adding the tabWidth along with the useTabs property, otherwise...would the code editor take the default one? 🤔

Reply:
Yep it will, but I think we prefer that because everyone has an opinion about how many spaces a tab should make up. If we leave a tab-width definition out of this file then each person can configure their editor however they want 🙂  this is the same for `dotcom-debug-user`

Reply:
Yep, maybe I did the same comment for dotcom-debug-user 😄